.github/workflows/release-workflow.yaml

```diff
@@ -0,0 +1,193 @@
+name: Main workflow
```

Comment:
Maybe put a more accurate name : release workflow

Reply:
This file will be deleted for now 😄  that was my mistake